### PR TITLE
updated the perfect scrollbar

### PR DESCRIPTION
**Pull request by Rohit25negi**
This is to update the perfect scrollbar which is breaking right now.
https://travis-ci.org/github/dimagi/commcare-hq/jobs/682861459

### CODE REVIEW

File: custom/icds_reports/templates/icds_reports/dashboard.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-datatables/dist/plugins/bootstrap/datatables.bootstrap.min.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'leaflet/dist/leaflet.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-busy/dist/angular-busy.min.css' %}"/>
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'perfect-scrollbar/css/perfect-scrollbar.min.css' %}"/>
+        <link type="text/css" rel="stylesheet" media="all" href="{% static 'utatti-perfect-scrollbar/css/perfect-scrollbar.css' %}"/>
```
Comment (Rohit25negi) [inline custom/icds_reports/templates/icds_reports/dashboard.html:18]: @orangejenny  is this fine to not have `min` I can't find the min version of it.

Comment (orangejenny) [reply custom/icds_reports/templates/icds_reports/dashboard.html:18]: Yeah, it's fine, it'll just add a tiny bit of time to dashboard load time.

**Review by orangejenny — APPROVED** [review]
Looks good. Are you planning to sanity test on india before this goes to TCL?

**Review by czue — APPROVED** [review]
embarassingly i don't even really know what this library does.

but confirming you either do, or clicked around dashboard enough to feel good about the upgrade/change?

**Review by kaapstorm — APPROVED** [review]
The build has got further than it used to, so it looks like this has resolved it.

**Comment by Rohit25negi** [comment]
> but confirming you either do, or clicked around dashboard enough to feel good about the upgrade/change?

Hey yeah, I think its used to scrollbar styling on left navigation we have. Yes tested this on local. Going to test on India before deploying.

**Comment by kaapstorm** [comment]
Third Travis job exceeded time limit :unamused: Restarted.

**Comment by Rohit25negi** [comment]
Tested this on India. Looks good.

**Pull request merged**